metrics/metrics.go

```diff
@@ -123,7 +123,10 @@ var (
 	)
 )
 
-func init() {
+// StartServer runs a metrics server inside a goroutine
+// that reports default application metrics in prometheus format.
+// Any errors that may occur will stop the server add log.Fatal the error.
+func StartServer() {
```

Comment:
I would also take the port into this function's arguments, so that this function doesn't rely on the environment to decide that.

Reply:
done